doc/computing/parallelism.rst

```diff
@@ -299,6 +299,13 @@ When this environment variable is set to a non zero value, the `Cython`
 derivative, `boundscheck` is set to `True`. This is useful for finding
 segfaults.
 
+`SKLEARN_BUILD_ENABLE_DEBUG_SYMBOLS`
+~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+
+When this environment variable is set to a non zero value, the debug symbols
+will be included in the compiled C extensions. Only debug symbols for POSIX
+systems is configured.
```

Comment:
this doesn't really belong to `parallelism.rst` though.

Reply:
As for discoverability, searching for "configuration" places [the `parallelism.rst` page as second](https://scikit-learn.org/stable/search.html?q=configuration). I'm open to figuring out a better location for all the configuration options, but that is another discussion.